packages/app/src/systems/Transaction/pages/ViewActivity/ViewActivity.tsx

```diff
@@ -0,0 +1,40 @@
+import { Stack } from '@fuel-ui/react';
+import { Address } from 'fuels';
+import { useNavigate } from 'react-router-dom';
+
+import { ActivityList } from '../../components/ActivityList/ActivityList';
+import { useTxs } from '../../hooks/useTxs';
+
+import { useAccounts } from '~/systems/Account';
+import { Layout } from '~/systems/Core';
+import { NetworkScreen, useNetworks } from '~/systems/Network';
+
+export function ViewActivity() {
+  const navigate = useNavigate();
+  const networks = useNetworks({ type: NetworkScreen.list });
+  const providerUrl = networks?.selectedNetwork?.url;
+  const { account, isLoading } = useAccounts();
+
+  const address = account
+    ? Address.fromAddressOrString(account?.address).toB256()
+    : '';
+  const { isLoadingTx, txs } = useTxs({
+    address,
+    providerUrl,
+  });
+
+  return (
+    <Layout title="Activity" isLoading={isLoading}>
+      <Layout.TopBar onBack={() => navigate(-1)} />
```

Comment:
Once navigating to the page the back button should return to wallet.

If you click multiple times on the Acitivity item you need to click multiple times before it returns to the wallet.